ComputeVisibleServices/function.json

```diff
@@ -0,0 +1,17 @@
+{
+ "scriptFile": "../lib/index.js",
+ "disabled": false,
+ "bindings": [
+    {
+      "type": "cosmosDBTrigger",
```

Comment:
reacts to cosmosdb change feed events for the services collection (create / update)